boards/lilygo/ttgo_t8c3/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Lothar Felten <lothar.felten@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 65535 if WIFI && BT
+	default 51200 if WIFI
+	default 40960 if BT
+	default 4096
+
+choice BT_HCI_BUS_TYPE
```

Comment:
This will/might conflict with https://github.com/zephyrproject-rtos/zephyr/pull/72323. Depends on merging order.

Reply:
@lfelten #72323 has been merged, please update accordingly